sihl/src/contract_queue.ml

```diff
@@ -71,7 +74,7 @@ let should_run (job_instance : instance) now =
 let default_tries = 5
 let default_retry_delay = Core_time.Span.minutes 1
 
-let default_error_handler msg (instance : instance) =
+let default_error_handler ?ctx:_ msg (instance : instance) =
   Lwt.return
   @@ Logs.err (fun m ->
```

Comment:
may add "pool" to log message if exists

```ocaml
let pool = 
  CCList.assoc_opt ~eq:(=) "pool"
  |> CCOption.map (fun pool -> Format.asprintf " (pool: '%s')" pool)
  |> CCOption.get_or ~default:""
```

or full context

```ocaml
(* with ppx_deriving.show *)
let context = Format.asprintf " (ctx: %s)" ([%show: (string * string) list] ctx) in
```

Reply:
Good point, thanks!